include/valijson/constraints/concrete_constraints.hpp

```diff
@@ -192,6 +192,11 @@ class ConstConstraint: public BasicConstraint<ConstConstraint>
       : BasicConstraint(other),
         m_value(other.m_value->clone()) { }
 
+    ~ConstConstraint() override
+    {
+        delete m_value;
```

Comment:
I don't entirely follow why this is required. `m_value` is a `std::unique_ptr`, so I would have expected the `FrozenValue` destructor to be called, and its memory to be freed, when the ConstConstraint class is destroyed.

Reply:
This actually doesn't compile. Is there something I'm missing here?

Reply:
Ah, I see, we are using an older version. In our version this change fixed the memory leak. I will close this pull request. Sorry for the hassle.